src/openml.jl

```diff
@@ -257,6 +257,77 @@ function load_List_And_Filter(filters::String; api_key::String = "")
     return nothing
 end
 
+qualitynames(x) = haskey(x, "name") ? [x["name"]] : []
+
+"""
+    list_datasets(filter = ""; api_key = "", output_format = NamedTuple)
+
+List OpenML datasets. See [`load_List_And_Filter`](@ref) for the format of
+the filter. As an alternative `output_format` one can choose other table types,
```

Comment:
`load_List_And_Filter` does not appear to be a method. Where is this from? Is this a reference to something at OpenML site itself? If so, perhaps a url reference would be more handy?